incubator-tools/tif_to_jpeg_image_conversion/tif_to_jpeg_image_conversion.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
<p>This seems like a very basic tool using base Python libraries. Maybe should just be in utilities? Or not included.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Removed the  notebook. Updated it as a function in the utilities.py